hack/deploy-logging.sh

```diff
@@ -150,75 +236,67 @@ wait_for_logging_is_running() {
 }
 
 deploy_logging_using_olm() {
-    # Create the $ESO_NS namespace:
-    if oc get project $ESO_NS > /dev/null 2>&1 ; then
-        echo using existing project $ESO_NS
-    else
-        oc create -f $TEST_OBJ_DIR/openshift-operators-redhat-namespace.yaml
-    fi
-    oc get projects | grep logging || :
-
-    # Create an OperatorGroup for $LOGGING_NS:
-    oc -n ${LOGGING_NS} create -f $TEST_OBJ_DIR/openshift-logging-operatorgroup.yaml
-    # Create an OperatorGroup for $ESO_NS:
-    oc -n $ESO_NS create -f $TEST_OBJ_DIR/openshift-operators-redhat-operatorgroup.yaml
-
-    # Create the CatalogSourceConfig for the elasticsearch-operator in the namespace openshift-marketplace:
-    oc create -n openshift-marketplace -f $TEST_OBJ_DIR/elasticsearch-catalogsourceconfig.yaml
-    oc get -n openshift-marketplace CatalogSourceConfig | grep elasticsearch || :
-
-    # Create the subscription for elasticsearch in the namespace $ESO_NS:
-    oc create -n $ESO_NS -f $TEST_OBJ_DIR/elasticsearch-subscription.yaml
-    oc get -n $ESO_NS subscriptions | grep elasticsearch || :
-
-    # Create the CatalogSourceConfig for cluster-logging in the namespace openshift-marketplace:
-    oc create -n openshift-marketplace -f $TEST_OBJ_DIR/cluster-logging-catalogsourceconfig.yaml
-    oc get -n openshift-marketplace CatalogSourceConfig | grep logging || :
-
-    # create the subscription in the namespace $LOGGING_NS:
-    oc create -n ${LOGGING_NS} -f $TEST_OBJ_DIR/cluster-logging-subscription.yaml
-    oc get -n ${LOGGING_NS} subscriptions | grep logging || :
+    local manifest=$( mktemp -d )
+    trap "rm -rf $manifest" RETURN
+
+    local eoimg=${EO_IMAGE:-$( construct_image_name elasticsearch-operator latest )}
+    cp -r ${EO_DIR}/manifests/${MASTER_VERSION} $manifest
+    cp ${EO_DIR}/manifests/*.package.yaml $manifest
+    update_images_in_clo_yaml $manifest/${MASTER_VERSION}/elasticsearch-operator.*.clusterserviceversion.yaml $eoimg
+    for pkg in $manifest/*.package.yaml ; do
+        sed -e 's/name: \([0-9.][0-9.]*\)$/name: "\1"/' -i $pkg
+    done
+    SUFFIX="-eo" \
+    CONFIGMAP_NAME=eo-olm \
+    NAMESPACE=${LOGGING_NS} \
+    VERSION=${MASTER_VERSION} \
+    OPERATOR_IMAGE=$eoimg \
+    MANIFEST_DIR=${manifest} \
+    TEST_NAMESPACE=${LOGGING_NS} \
+    TARGET_NAMESPACE=${LOGGING_NS} \
+    hack/vendor/olm-test-script/e2e-olm.sh
+
+    local cloimg=${CLO_IMAGE:-$( construct_image_name cluster-logging-operator latest )}
```

Comment:
are not all the images available to us via environment vars injected by the CI  in the format of `IMAGE_COMPONENT` like `IMAGE_CLUSTER_LOGGING_OPERATOR` or `IMAGE_ELASTICSEARCH_OPERATOR`

Reply:
They are available.  `IMAGE_FORMAT` looks like this: `registry.svc.ci.openshift.org/ci-op-xxx/stable:${component}` - so there will be `registry.svc.ci.openshift.org/ci-op-xxx/stable:logging-elasticsearch5`, `registry.svc.ci.openshift.org/ci-op-xxx/stable:logging-ci-test-runner`, etc.